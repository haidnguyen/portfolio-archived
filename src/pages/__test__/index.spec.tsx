import { render } from '@testing-library/react';
import Home from 'pages';
import React from 'react';

describe('Home', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Home></Home>);

    expect(asFragment()).toMatchSnapshot();
  });
});
