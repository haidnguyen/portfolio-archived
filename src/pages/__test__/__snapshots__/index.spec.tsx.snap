// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<DocumentFragment>
  <header
    class="h-screen"
  >
    <div
      class="h-full relative flex justify-center items-center"
    >
      <canvas
        class="h-full screen-canvas"
        id="canvas"
      >
        Your browser does not support HTML5 canvas
      </canvas>
      <div
        class="absolute flex flex-col text-4xl justify-center items-center text-white"
      >
        <div
          class="flex flex-col"
        >
          <span>
            Hi, I‘m 
            <span
              class="text-name"
            >
              Nguyễn Đình Hải
            </span>
          </span>
          <span>
            Full-Stack Web Developer
          </span>
        </div>
        <div
          class="main-button font-medium"
        >
          <span
            class="mr-3"
          >
            View my work
          </span>
          <svg
            class="main-arrow"
            fill="none"
            height="20"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
            <polyline
              points="12 5 19 12 12 19"
            />
          </svg>
        </div>
      </div>
    </div>
  </header>
  <main>
    Body
  </main>
  <footer>
    Footer
  </footer>
</DocumentFragment>
`;
